autosubmit/autosubmit.py

```diff
@@ -4521,7 +4521,7 @@ def create(expid, noplot, hide, output='pdf', group_by=None, expand=list(), expa
                     as_conf.reload(force_load=True,only_experiment_data=True)
                     # Getting output type provided by the user in config, 'pdf' as default
                     try:
-                        if not Autosubmit._copy_code(as_conf, expid, as_conf.experiment_data.get("PROJECT",{}).get("PROJECT_TYPE","none"), False):
+                        if not Autosubmit._copy_code(as_conf, expid, as_conf.get_project_type(), False):
```

Comment:
There are probably other parts of the code that can replace dictionary access in `as_conf` by functions, like this one.